### PR TITLE
chore(doc): doc to configure volume-event-exporter to export events

### PR DESCRIPTION
**Pull request by mittachaitu**
This PR add doc to explain configuration of volume-event-exporter
to export volume(create, delete) events to external service.

Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>

### CODE REVIEW

**Comment by mittachaitu** [comment]
This can be merged after merging https://github.com/openebs/dynamic-nfs-provisioner/pull/93

**Review by mynktl — APPROVED** [review]

**Pull request merged**
